readme.md

```diff
@@ -1,4 +1,4 @@
-FlatBuffers Version 1.0
+FlatBuffers Version 1.0 [![Build Status](https://travis-ci.org/franramirez688/flatbuffers.svg?branch=master)](https://travis-ci.org/franramirez688/flatbuffers)
```

Comment:
If we're going to setup Travis for Flatbuffers we should probably have it associated with a Flatbuffers maintainer's account instead.